src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Abs.txt

```diff
@@ -17,6 +23,12 @@
 >> Abs(-2)
 2
 
+>> Abs(-2147483649) // testing a small number
```

Comment:
This is still a very large number - a large _negative_ number. Typically "small numbers" represent those close to zero (e.g., 1e-23, 0.000000000321, and so on). A better comment would be something like you had before, which explains why you chose this number specifically. Tests can be seen as the _de facto_ specification for a product, and you made a conscious choice in choosing this value (+/- 2^31+2), so it's useful to let the reader know why this is the case.

Reply:
Same for the other cases (2^53-1) - there is a reason why we chose 9007199254740991 and not 1234567890123456.